pkg/controller/job/job_controller_test.go

```diff
@@ -1860,8 +1858,9 @@ func TestSyncPastDeadlineJobFinished(t *testing.T) {
 	}
 	for _, tc := range tests {
 		t.Run(tc.name, func(t *testing.T) {
+			jobKey := fmt.Sprintf("default/%s", tc.jobName)
```

Comment:
Use a helper method `controller.FuncKey` to get the key (but preferably as mentioned about keep the fix minimal if possible).

Reply:
/Ack